Documentation/files/logging/fluentd-configmap.yaml

```diff
@@ -213,6 +206,13 @@ data:
     </source>
 
   kubernetes-filter.conf: |
+    # Prevent fluentd from handling records containing its own logs. Otherwise
+    # it can lead to an infinite loop, when error in sending one message generates
+    # another message which also fails to be sent and so on.
+    <match kubernetes.var.log.containers.fluentd**>
+      type null
+    </match>
+
```

Comment:
Prevent fluentd from handling records containing fluentd logs. Allowing this access can lead to an infinite loop, in which an error in sending one message generates another message which also fails to be sent, and so on.